libs/net/dal/Services/ReportService.cs

```diff
@@ -134,9 +134,9 @@ public IEnumerable<Report> FindMyReports(int userId)
     /// </summary>
     /// <param name="id"></param>
     /// <param name="ownerId">The owner of the instance.</param>
-    /// <param name="isSent">Whether to get the instance that was sent.</param>
+    /// <param name="limit">Number of instances to return.</param>
     /// <returns></returns>
-    public ReportInstance? GetLatestInstance(int id, int? ownerId = null, bool? isSent = null)
+    public IEnumerable<ReportInstance> GetLatestInstances(int id, int? ownerId = null, int limit = 2)
```

Comment:
We now return the most recent two instances so that we can also see the last `SentOn` value.